src/ukify/ukify.py

```diff
@@ -536,50 +538,95 @@ def pe_add_sections(uki: UKI, output: str):
 
     pe.write(output)
 
+def signer_sign(cmd):
+    print('+', shell_join(cmd))
+    subprocess.check_call(cmd)
 
-def make_uki(opts):
-    # kernel payload signing
+def find_sbsign(opts=None):
+    return find_tool('sbsign', opts=opts)
 
-    sbsign_tool = find_tool('sbsign', opts=opts)
-    sbsign_invocation = [
-        sbsign_tool,
-        '--key', opts.sb_key,
-        '--cert', opts.sb_cert,
+def sbsign_sign(sbsign_tool, input_f, output_f, opts=None):
+    sign_invocation = [
+            sbsign_tool,
+            '--key', opts.sb_key,
+            '--cert', opts.sb_cert,
+            input_f,
+            '--output', output_f,
+        ]
+    if opts.signing_engine is not None:
+        sign_invocation += ['--engine', opts.signing_engine]
+    signer_sign(sign_invocation)
+
+def find_pesign(opts=None):
+    return find_tool('pesign', opts=opts)
+
+def pesign_sign(pesign_tool, input_f, output_f, opts=None):
+    sign_invocation = [
+        pesign_tool, '-s', '--force',
+        '-n', opts.sb_certdir,
+        '-c', opts.sb_cert_name,
+        '-i', input_f,
+        '-o', output_f,
     ]
+    signer_sign(sign_invocation)
 
-    if opts.signing_engine is not None:
-        sbsign_invocation += ['--engine', opts.signing_engine]
+sbverify = {
```

Comment:
```suggestion
SBVERIFY = {
```
PEP8 recommends capital names for [constants](https://peps.python.org/pep-0008/#constants)